modules/security-jwt/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+#security.authentication.keystore.keystorelocation=C:/Devon-dist_3.0.0/jwtkeystore.p12
+#security.authentication.keystore.keystoretype=PKCS12
+#security.authentication.keystore.password=123456
```

Comment:
For passwords there is never a reasonable default. This should be a secret that differes per environment. So this config should only go to `config/application.properties` but shall be removed here.

Reply:
https://github.com/devonfw/devon4j/blob/develop/documentation/guide-configuration.asciidoc#externalized-configuration